e2e/tests/plugins/timeConductor/timeConductor.e2e.spec.js

```diff
@@ -23,6 +23,13 @@
 const { test } = require('../../../fixtures.js');
 const { expect } = require('@playwright/test');
 
+const SELECTOR = {
+    START_OFFSET_BUTTON: 'data-testid=conductor-start-offset-button',
```

Comment:
I think with the selectors as descriptive as they are, we won't need to declare them :P

Reply:
I took it too far! :) I'll revert this